### PR TITLE
What data is not protected via customer key

### PR DESCRIPTION
**Pull request by joinimran**
As per user recommendation got by MS support, I have added another query whereas it mentioned what data is not protected by customer key.

Problem: https://github.com/MicrosoftDocs/microsoft-365-docs/issues/634

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/compliance/service-encryption-with-customer-key-faq.md [hunk 1]
```diff
@@ -30,6 +30,11 @@ Customer Key enhances the ability of your organization to meet the demands of co
 <a name="WhatDataIsCoveredbyCustomerKey"> </a>
 
 SharePoint Online site content and the files stored on that site and files uploaded to OneDrive for Business are covered. Exchange Online mailbox content (e-mail body, calendar entries, and content of email attachments) is covered. Text conversations from Skype for Business are covered, but Skype Meeting Broadcast recordings and Skype Meeting content uploads are not covered. Skype Meeting Broadcast and Skype Meeting content uploads are encrypted along with all other content in Office 365, but we currently don't offer customer control of the encryption keys.
+
+## What Office 365 data at rest is not covered by Customer Key?
+<a name="WhatDataIsNotCoveredbyCustomerKey"> </a>
+
+Customer Key is only used to encrypt files, it does not encrypt any other SharePoint Online content such as metadata in documents and libraries, data in lists except attachments, data stored in user profile, data stored in search index, data stored in other services such as Planner, Stream, Delve, Microsoft Teams, Yammer etc.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/service-encryption-with-customer-key-faq.md]:
```suggestion
Customer Key is only used to encrypt files. It does not encrypt any other SharePoint Online content such as metadata in documents and libraries, data in lists except attachments, data stored in user profiles, data stored in search indices, and data stored in other services such as Planner, Stream, Delve, Microsoft Teams, Yammer, etc.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy Review

**Comment by joinimran** [comment]
Ready to go live

**Comment by mypil** [comment]
@KCCross - PR has been copy edited and is ready for final review based on issue #634. 

Thank you.

cc: @kenwith

**Comment by joinimran** [comment]
@KCCross , A followup on this PR. Did you get a chance to review the PR? Its already been copy-edited.

**Comment by mypil** [comment]
@KCCross @LauraWi - Can anyone of you please assist to review and merge this PR?

Thank you.

**Comment by LauraWi** [comment]
@KC Cross<mailto:KC.Cross@microsoft.com> – does this need PM review?

**Comment by KCCross** [comment]
The content in this PR is not entirely accurate. Reviewing with PM before rolling out update.

**Comment by mypil** [comment]
@KCCross - Do you have any updates on this? 
Please let us know if there are any changes needed for this PR so we can let @joinimran make the change. 

Thank you.

**Comment by mypil** [comment]
@KCCross - We would like to follow up. Any updates on this?

Thank you.

**Review by MSFTTracyP — CHANGES_REQUESTED** [review]
Also, cached content, any web part data, solutions, and other SharePoint customizations. Also, files synced down to a desktop or mobile device. Basically, Customer Key works for SPO, EXO, Teams (if you're using SPO for data storage), and some Skype for Business features (yes, for texts, no for recordings, and content uploads). Be aware, for the most part, we document what Customer Key covers, not what it does not cover.

**Comment by joinimran** [comment]
@MSFTTracyP You are suggesting to make this change as 

Customer Key is only used to encrypt files, cached content, any web part data, solutions, and other SharePoint customizations. It is also used for files synced down to a desktop or mobile device. Customer Key works for SharePoint Online, Exchange Online, Team (if you are using SharePoint Online for data storage) and some Skype for business features.

**Comment by MSFTTracyP** [comment]
@joinimran - Ohp. Teams. Not Team. To clarify, it _isn't_ used for files synced down to the desktop or mobile devices, cached content, web part data, solutions, and other SharePoint customizations. I'm sorry that was unclear. I was listing additional things it doesn't encrypt. It _is_ used for SharePoint Online, Exchange Online, Teams (if you are using SharePoint Online for data storage) and some Skype for Business features.

**Comment by joinimran** [comment]
@MSFTTracyP I got your point here that means we are going with a statement **It is used for SharePoint Online, Exchange Online, Teams (if you are using SharePoint Online for data storage) and some Skype for Business features.** Not with any other details like it doesn't work with ... as you have already mentioned that we tell it works with, not where it will not work. Correct?

**Comment by KCCross** [comment]
Hello everyone,
I’ve returned from vacation and from wide-ranging power outages in my area. I’ll touch base with Tracy shortly and pick up where we left off. Thanks,
KC

**Comment by MSFTTracyP** [comment]
@joinimran - So here's the rub. The question isn't 'What is protected by customer key?' it's 'What data is not protected via customer key?' In my opinion, we should always try to stick with the more concise 'what _is_ protected by Customer Key' info, as specifically as we can. But if the FAQ question is ~ what's not protected, and we can't ignore that question. The problem with the question is that I can say 'items in your purse' and be correct. When we answer this, we should acknowledge the scope of the question, and say that information about what _is_ protected will be more fleshed-out, and should be where customers' turn, first.

KC may choose to change the question in the FAQ if she feels this is needed, but there is value in letting customers know what doesn't fall under Customer Key and where they may want to look for further technologies (like device-based conditional access policies, and Azure Information Protection, both of which you can mention and link to) to safeguard their data. In other words, we'll answer the question. But we should conclude the question with a clear statement about what Customer Key _does_ protect.

**Comment by KCCross** [comment]
As luck would have it, the FAQ is going away entirely. We’re replacing the question in the FAQ with a comprehensive overview of customer key and how it compares to related technologies, complete with links to those technologies. I’m working with Tracy to ensure we catch the bit we missed in the original FAQ. Thank you Tracy for all you’ve captured here. This will surely improve the rewrite.

**Comment by joinimran** [comment]
@KCCross This means this document is going to be re-written? What inputs are required from my side? Or we can close this PR and issue with feedback that we are re-writing the document with all the required and updated info.

**Comment by mypil** [comment]
@KCCross - We would like to follow up and get your recommendation to be able to  close issue #634. Can you please advise how you want to proceed with this? 

Thank you.

**Comment by MSFTTracyP** [comment]
@mypil Hi! I know you're busy, but could you put together an updated response to the question given the information I've given you and place it in the document so that it can be used as reference material for KC's overview when the time comes for to publish it?

**Comment by mypil** [comment]
@MSFTTracyP - I received your email. It would be freelancer @joinimran who can provide more information about this PR. We can discuss this further internally with him via email.  Thanks.

**Comment by KCCross** [comment]
Apologies for the late comment here - The entire content area was being overhauled for the last several months. This delayed fixes to the existing content that weren't p0 breakfix. Since the Customer Key content is sensitive in nature, I found it prudent to wait until the entire new content set was available before closing these. Thank you for your understanding. I look forward to your feedback on the new content.

**Pull request closed without merging**